blackbox/reset-upstream-stratis-repos.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+STRATISD_REPO="https://github.com/stratis-storage/stratisd.git"
+STRATIS_CLI_REPO="https://github.com/stratis-storage/stratis-cli.git"
+
+STRATISD_N=$(rpmspec -P stratisd.spec | grep ^Name | awk {'print $2'})
```

Comment:
Do you really need to make the directories dependent on the information extracted from the rpm? That would only make sense if you had multiple different versions. But I think we have only one version, which you chose to be an extremely high number. So maybe you could just make the directories for stratisd and stratis-cli some standard name, like ```STRATISD```, ```STRATIS_CLI``` and not have to interrogate the rpm at all just for this purpose? The reason I propose this is that I'm worried about these duplicate lines, ```STRATISD_N```, etc. showing up in both files, and I don't think that's really necessary.

Reply:
I believe I do need to make the directories using the name-version-release name in the rpm spec, since that's how rpmbuild builds the packages.

Reply:
That does make me think of this question: could the part of the scripting that "pre-stages" the repositories leave their names in the plain form (e.g.: "stratisd", "stratis-cli"), to then be transformed by the packaging script?  At that point, we could reduce the number of scripts that use those variables.

We could avoid repetition by having these all be functions of one big script, but the utility of calling separate stages at different times would be lost.

Reply:
The delete phase needs to look up the old NVR name, from when the directory was renamed on the previous run, therefore it does need to define those variables.